course/style.css

```diff
@@ -3,7 +3,7 @@ Theme Name: Course
 Author: Automattic
 Author URI: https://automattic.com/
 Description: Course is a flexible and modern education theme for anyone wanting to share their knowledge. The theme is built with integration with Sensei LMS and is ideal for Sensei users that are creating or selling courses. Style variations with multiple font and color combinations help you craft the perfect look and feel to show off courses and content. The theme can be used without Sensei too.
-Version: 1.3.5
+Version: 1.3.7
```

Comment:
This should be 1.3.6, no?

Reply:
I thought so too initially, but this PR bumped all the version numbers including Course's

Reply:
It didn't trigger a deploy - https://wordpress.org/themes/course/. And I just spun up a new WP.com site and it's also on 1.3.5. Let's update the change log and version number to 1.3.6.

Reply:
Interesting 🤔 I didn't know it was possible to merge into trunk and not trigger a release here. I've updated the version number here https://github.com/Automattic/themes/pull/7692/commits/e52430fe6cc45c48cf7a0da59dcfed2372236eae . Thank you!